source/github-helpers/get-pr-info.ts

```diff
@@ -8,15 +8,27 @@ type PullRequestInfo = {
 	// https://docs.github.com/en/graphql/reference/enums#mergeablestate
 	mergeable: 'CONFLICTING' | 'MERGEABLE' | 'UNKNOWN';
 	viewerCanEditFiles: boolean;
+	headRef: {
+		compare: {
+			status: 'BEHIND' | 'DIVERGED' | 'AHEAD' | 'IDENTICAL';
+		};
+	};
 };
 
-export default async function getPrInfo(number = getConversationNumber()!): Promise<PullRequestInfo> {
+export default async function getPrInfo(base: string, number = getConversationNumber()!): Promise<PullRequestInfo> {
 	const {repository} = await api.v4(`
 		repository() {
 			pullRequest(number: ${number}) {
 				baseRefOid
 				mergeable
 				viewerCanEditFiles
+				headRef {
+					compare(headRef: "${base}") {
+						status
+						behindBy
+						aheadBy
```

Comment:
I put this here in case you wanted to do https://github.com/refined-github/refined-github/issues/3863#issuecomment-1192756619

Reply:
Indentation needs some fixing

Reply:
?
What are you referring to?

Reply:
Ah sorry I must have commented on the wrong thing: look for the `repository()` line that is indented one time too many.

Reply:
![image](https://user-images.githubusercontent.com/1402241/198860251-7a5b8918-e130-4341-aa1b-67a4545617f3.jpeg)